code/modules/cargo/packs.dm

```diff
@@ -559,6 +567,16 @@
 	crate_type = /obj/structure/closet/crate/secure/plasma
 	dangerous = TRUE
 
+/datum/supply_pack/security/armory/miniguns
+	name = "Personal Miniature Energy Guns"
+	desc = "Contains three miniature energy guns. Each gun has a dislaber and a lethal option. Requires Armory access to open."
+	cost = 5000
+	contains = list(/obj/item/gun/energy/e_gun/mini,
+					/obj/item/gun/energy/e_gun/mini,
+					/obj/item/gun/energy/e_gun/mini)
+	crate_name = "personal engery guns crate"
+	crate_type = /obj/structure/closet/crate/secure/plasma
```

Comment:
shouldn't this be a security crate

Reply:
Other Eguns are all in plasma crates lore wise do to being plasma based weaponds